pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+testpaths=tests
```

Comment:
A doc generation tool introduced into our examples submodule a few months ago uses pytest and its own `conftest.py` file to execute.

When trying to run the unit tests on this project, pytest was discovering that `conftest.py` as well as the one for this test suite and complaining about multiple conf files.

This `pytest.ini` configuration files tests pytest to ONLY look in the `tests` folder when executing the test suite (via our `make test` step).

Reply:
Thanks for adding the note to it!